sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering-validation.models.ts

```diff
@@ -614,6 +642,188 @@ export class OfferingValidationModel {
+    },
+  )
+  @IsEnum(OnlineInstructionModeOptions, {
+    message: getEnumFormatMessage(
+      userFriendlyNames.onlineInstructionMode,
+      OnlineInstructionModeOptions,
+    ),
+  })
+  onlineInstructionMode?: OnlineInstructionModeOptions;
+
+  /**
+   * Specifies if the blended offering will always be provided with the same total duration of online delivery.
+   * Values can be "yes" or "no".
+   */
+  @ValidateIf(
+    // Validate when the property must have a value or it already has a value.
+    (offering: OfferingValidationModel) =>
+      (offering.institutionContext?.isBCPrivate &&
+        offering.offeringDelivered == OfferingDeliveryOptions.Blended) ||
+      !!offering.isOnlineDurationSameAlways,
+  )
+  // Allow the property only when the institution either BC Public or BC Private and the offering is blended.
+  @AllowIf(
+    (offering: OfferingValidationModel) =>
+      (offering.institutionContext?.isBCPrivate ||
+        offering.institutionContext?.isBCPublic) &&
+      offering.offeringDelivered == OfferingDeliveryOptions.Blended,
+    undefined,
+    {
+      message: `${userFriendlyNames.isOnlineDurationSameAlways} is not allowed for provided institution type or offering delivery type and offering online delivery inputs.`,
+    },
+  )
+  @IsEnum(OfferingYesNoOptions, {
+    message: getEnumFormatMessage(
+      userFriendlyNames.isOnlineDurationSameAlways,
+      OfferingYesNoOptions,
+    ),
+  })
+  isOnlineDurationSameAlways?: OfferingYesNoOptions;
+
+  /**
+   * Percentage of total duration that will be provided by online delivery in the blended offering.
+   */
+  @ValidateIf(
+    // Validate when the property must have a value or it already has a value.
+    (offering: OfferingValidationModel) =>
+      (offering.institutionContext?.isBCPrivate &&
+        offering.offeringDelivered == OfferingDeliveryOptions.Blended &&
+        offering.isOnlineDurationSameAlways === OfferingYesNoOptions.Yes) ||
+      (offering.totalOnlineDuration !== null &&
+        offering.totalOnlineDuration !== undefined),
+  )
+  // Allow the property only when the institution either BC Public or BC Private and the offering is blended
+  // and the online duration is same always.
+  @AllowIf(
+    (offering: OfferingValidationModel) =>
+      (offering.institutionContext?.isBCPrivate ||
+        offering.institutionContext?.isBCPublic) &&
+      offering.offeringDelivered == OfferingDeliveryOptions.Blended &&
+      offering.isOnlineDurationSameAlways === OfferingYesNoOptions.Yes,
+    undefined,
+    {
+      message: `${userFriendlyNames.totalOnlineDuration} is not allowed for provided institution type or offering delivery type and offering online delivery inputs.`,
+    },
+  )
+  @IsNumber()
+  @Min(1, {
+    message: getMinFormatMessage(userFriendlyNames.totalOnlineDuration),
+  })
+  @Max(99)
```

Comment:
These numbers should be converted to const following the same from the other `@Max` validation in the class.